src/Oscoin/Node/Trans.hs

```diff
@@ -48,12 +50,18 @@ runNodeT env (NodeT ma) = runReaderT ma env
 
 -- | Node static config.
 data Config = Config
-    { cfgEnv             :: Environment
+    { cfgGlobalConfig    :: GlobalConfig
     , cfgTelemetry       :: Telemetry.Handle
     , cfgNoEmptyBlocks   :: Bool
     , cfgConsensusConfig :: Consensus.Config
     }
 
+data GlobalConfig = GlobalConfig
+    { globalEnv             :: Environment
+    , globalLogicalNetwork  :: Network
+    , globalPhysicalNetwork :: P2P.Network
+    }
+
```

Comment:
Any reason why we have a record for this, vs. having the fields directly in `Config`? What is it that makes this config 'global' and the other not? Is it because it's network-wide?

Reply:
Perhaps this can be answered in a haddock :man_shrugging:

Reply:
idk, can't decide where to put this - the types are defined in the "global" `Oscoin.Configuration` module, and I expect them to be threaded to other places in the code (irrespective of the `Node`'s config).

Reply:
It _should_ probably be in `Oscoin.Configuration` with ctor hidden as not all combinations make sense / should be allowed. Can't import the P2P types though, because cycle.